src/test/scala/me/jeffmay/util/TestHooks.scala

```diff
@@ -0,0 +1,6 @@
+package me.jeffmay.util
+
+// TODO: Use this to setup / tear down the database / connections
+object TestHooks {
+
+}
```

Comment:
While the `TestCleanup` / `TestInitialization` pattern is cool. I decided I could put it all together into a `TestHooks` object and have cleanup and shutdown classes for sbt to load before and after all tests. I never started or finished this effort, so this is the shell of that work. Might still be good to do if I have time in this PR.